packages/react-components/react-drawer/.eslintrc.json

```diff
@@ -1,4 +1,13 @@
 {
   "extends": ["plugin:@fluentui/eslint-plugin/react"],
-  "root": true
+  "root": true,
+  "overrides": [
```

Comment:
Override to allow us to have shared tests to be placed on the e2e directory. This allows us to create a function encapsulating base scenarios that can be called for the Drawer, DrawerInline and DrawerOverlay.